### PR TITLE
Tmb

### PR DESCRIPTION
**Pull request by jonca79**

### CODE REVIEW

**Comment by Smeds** [comment]
Jenkins fail since the test config file are missing TMB entry

**Comment by Smeds** [comment]
Test config files are missing GATK_CNV_SNPs entrt

File: src/scripts/python/TMB.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+output_tmb = open(snakemake.putput.tmb, "w")
+
+
+FFPE_SNV_artifacts = {}
+header = True
+for line in artifacts:
+    if header:
+        header = False
+        continue
+    lline = line.strip().split("\t")
```
Comment (Smeds) [inline src/scripts/python/TMB.py]: maybe skip first line by reading it before the loop
```
next(artifacts)
for line in artifacts:
    lline = line.strip().split("\t")
```

File: src/scripts/python/TMB.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+nr_TMB = 0
+header = True
+prev_pos = ""
+prev_chrom = ""
+for line in vcf:
+    if header:
+        if line[:6] == "#CHROM":
+            header = False
+        continue
+    lline = line.strip().split("\t")
```
Comment (Smeds) [inline src/scripts/python/TMB.py]: maybe skip first line by reading it before the loop
```
next(vcf)
for line in artifacts:
    lline = line.strip().split("\t")
```

**Review by Smeds — CHANGES_REQUESTED** [review]

File: src/scripts/python/TMB.py (hunk 1 above)
Comment (jonca79) [reply src/scripts/python/TMB.py]: Fixed

File: src/scripts/python/TMB.py (hunk 2 above)
Comment (jonca79) [reply src/scripts/python/TMB.py]: Will not work as there are several header lines.

Comment (Smeds) [reply src/scripts/python/TMB.py]: true

**Review by Smeds — APPROVED** [review]

**Pull request merged**